discord/webhook/sync.py

```diff
@@ -290,8 +291,9 @@ def execute_webhook(
         files: Optional[Sequence[File]] = None,
         thread_id: Optional[int] = None,
         wait: bool = False,
+        has_view: bool = False,
```

Comment:
```suggestion
        with_components: bool = False,
```

Do this everywhere else.